common/http/src/client.rs

```diff
@@ -43,6 +44,8 @@ impl Client {
     pub fn new(
         template: RequestTemplate,
         state_handles: Option<(FileOffsetWriteHandle, FileOffsetFlushHandle)>,
+        retry_base_delay: Duration,
```

Comment:
This is much better, even if we end up hard coding it in the config and never override it